README.md

```diff
@@ -1,8 +1,3 @@
-<head>
-  <base href="https://github.com/GoogleCloudPlatform/django-cloud-deploy/blob/master/"
```

Comment:
Does this not fix the problem with relative links?

Reply:
Unfortunately not, see https://pypi.org/project/django-cloud-deploy/

Reply:
OK, thanks!